$NetBSD: patch-ad,v 1.3 2013/11/24 05:53:52 dholland Exp $

- use (semi-)standard headers
- fix os conditionals (apparently with respect to VMS)
- don't declare own errno

--- data.h.orig	1999-02-24 11:04:50.000000000 +0000
+++ data.h
@@ -24,6 +24,8 @@
  * SOFTWARE.
  */
 
+#include <sys/types.h>
+
 extern XPoint VTbox[];
 extern XPoint hVTbox[];
 extern Char *bptr;
@@ -34,7 +36,7 @@ extern char *ptydev;
 extern char *ttydev;
 extern char *xterm_name;
 extern Char buffer[];
-#if defined(linux) || defined(SVR4)||(defined(hpux)&&OSMAJORVERSION>=10)
+#ifndef VMS
 extern fd_set Select_mask;
 #else
 extern int Select_mask;
@@ -45,7 +47,6 @@ extern int bcnt;
 #ifdef DEBUG
 extern int debug;
 #endif	/* DEBUG */
-extern int errno;
 extern int max_plus1;
 extern int pty_mask;
 extern int switchfb[];
